amo2kinto/importer.py

```diff
@@ -28,6 +29,11 @@ def sync_records(amo_records, fields,
                  kinto_client, bucket, collection, schema, permissions):
 
     amo_records = prepare_amo_records(amo_records, fields)
+
+    if schema:
+        for record in amo_records:
+            jsonschema.validate(record, schema)
```

Comment:
That's the question I was asking myself, in anycase it shouldn't fails or it means either the AMO database is broken (which is the case on the dev server for instance) either the schema is not up-to-date.

To answer this question we should answer: Why do we want local validation?

- Is it to make sure the kinto database only contains valid records?
- Is it to make sure records we read from AMO are valids?